web/app/themes/mitlib-parent/inc/content-shortcrumb.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * The template used for displaying page content for pages in the shortrumb category in page-standard.php
```

Comment:
Possible typo `shortrumb` is maybe `shortcrumb`?

Reply:
Yeah, that's a typo. The line of code doesn't surface anywhere in the application, however (unlike things like the `Template Name: Foo` lines, which do get re-presented in the back end) - so I'm not going to worry about it too much yet.